### PR TITLE
TLV meetup - 7th birthday

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

File: _data/events.yml [hunk 1]
```diff
@@ -362,3 +362,18 @@
...
+  end: 2016-07-21 20:00 IST
+  url: 'https://www.eventbrite.com/e/foreman-112-release-party-7th-birthday-tickets-26418317900'
+  description: |
+    Please join us and celebrate Foreman 7th. birthday and 1.12 release.
+    Agenda:
+    * Foreman in 7 years perspective - Ohad Levy
+    * Metal-As-A-Service and discovery plugin - Ori Rabin
+    * Chef ❤ Foreman - Alex Balk
+    * Automating OpenSCAP reports with Foreman - Shlomi Zadok
+    * ReX demo
```
Comment (ohadlevy) [inline _data/events.yml]: does it make sense to add links (e.g. chef plugin, scap plugin etc)? also not sure if people know ReX

Comment (shlomizadok) [inline _data/events.yml]: ReX is a much better name than `remote execution` :gun: :)
Will add links

**Comment by GregSutcliffe** [comment]
Can you add it to the blog post as well?

**Comment by shlomizadok** [comment]
@GregSutcliffe - Added the blog post too. Thanks :)

**Comment by GregSutcliffe** [comment]
Merged as 757759f, ty @shlomizadok!

**Pull request closed without merging**